filters/auth/jwt_metrics_test.go

```diff
@@ -223,7 +225,44 @@ func TestJwtMetrics(t *testing.T) {
 			`,
 			request: &http.Request{Method: "GET", Host: "foo.test",
 				Header: http.Header{"Authorization": []string{
-					"Bearer foobarbaz",
+					validAuthHeader,
+				}},
+			},
+			status: http.StatusOK,
+			expected: map[string]int64{
+				"jwtMetrics.custom.GET.foo_test.200.invalid-token": 1,
+			},
+			expectedTag: "invalid-token",
+		},
+		{
+			name:     "no metrics when host matches opted-out domain",
+			filters:  `jwtMetrics("{issuers: [foo, bar], optOutHosts: [ '^.+[.]domain[.]test$', '^exact[.]test$' ]}")`,
+			request:  &http.Request{Method: "GET", Host: "foo.domain.test"},
```

Comment:
Should we test the same filter, but with "domain.test" host?

Reply:
We can but the goal is not to test regexp itself or all edge cases but the logic when host matches any or none of the configured patterns.